tekton/release-pipeline.yaml

```diff
@@ -54,14 +54,6 @@ spec:
     - name: git-clone
       taskRef:
         name: git-clone
-        resolver: bundles
-        params:
-          - name: bundle
-            value: gcr.io/tekton-releases/catalog/upstream/git-clone:0.9
-          - name: name
-            value: git-clone
-          - name: kind
-            value: task
```

Comment:
Why don't we remove line 56 and keep the remote resolution approach?

Reply:
maybe use v0.7 https://github.com/tektoncd/catalog/tree/main/task/git-clone/0.7 which doesn't need security context?

Reply:
done